extensions/toggl-track/src/index.tsx

```diff
@@ -90,6 +122,22 @@ function ListView() {
                           onSubmit={() => resumeTimeEntry(timeEntry)}
                           icon={{ source: Icon.Clock }}
                         />
+                        <PushAction
```

Comment:
I really love that editing/deleting the current running time entry has been added. But I don't think it makes sense to be able to delete/edit previous entries as the extension is set up currently. The entries shown in the list are not representative of _all_ entries, but only unique groups of name+pid. In my workflow I often create multiple time entries with the same name and project id. So in my case, which time entry exactly is deleted? I suggest to remove the ability to delete and edit previous time entries, but keep the possibility to edit and delete the running time entry.

We could try to make the list of time entries more like on the Toggl Track web edition. But as of now this extension is not ready for edit/delete of time entry history

Reply:
I agree, It currently deletes the most recent time entry of the group, which has been useful for me for some quick deletes, but I've also had to go onto the site to delete things at times.